app/src/main/java/org/p2p/wallet/receive/eth/ReceiveEthereumFragment.kt

```diff
@@ -62,6 +62,7 @@ class EthereumReceiveFragment :
                 showUiKitSnackBar(messageResId = R.string.receive_eth_address_copied)
             }
             financeBlockAddress.apply {
+                updateHorizontalPadding(start = 16, end = 8)
```

Comment:
what the... it should not be like that

it should be a common design

Reply:
![Screenshot 2023-03-12 at 23 03 42](https://user-images.githubusercontent.com/122359284/224560303-f030a6bc-b425-4e2d-9d83-ac9a03e4e745.png)

Reply:
we have 24 px space on right side, that's why I add this method x D

Reply:
add `app:layout_goneMarginEnd="0dp"` to `@+id/leftSideView` in widget_finance_block.xml